### DISCUSSION TITLE
Page alias

### DISCUSSION THREAD

**Discussion by Portulak-3013** [athensresearch/athens#1326] [Feature Requests]
Alias for a page: `[alias]([[page]])`

**EDIT: aliases were implemented this way: https://github.com/athensresearch/athens/discussions/1326#discussioncomment-923738.**

**Comment by juniusfree** [comment]
Also, alias for a block: `[alias](((block-id))))`

**Comment by juniusfree** [comment]
Or maybe a full support for Markdown?

**Comment by juniusfree** [comment]
https://discord.com/channels/708122962422792194/800442937602342922/819461273833046046

**Comment by sid597** [comment]
This is discussed here also https://github.com/athensresearch/athens/discussions/1001 , seeing the no of reactions I think it would be best to keep this open, but discussions would be a better place for this.

**Comment by neotyk** [comment]
Proposed Aliases approach.

I don't like the amount of brackets you'd use with both approaches that ROAM offers.

What if we instead would extend our Athens Flavoured Markdown with following:
* `[alias text]((block-uid))` for block aliases
* `[alias text][[Page Title]]` for page aliases

This is an extension of Markdown links `[title](url)` which takes `[title]` part to provide alias text and replaces `(url)` with either `((block-ref))` or `[[page link]]`.

Yey 👍  or Ney 👎 ?
Vote with reaction to this comment.

If you see a cleaner way to do aliases, please share.

**Reply by Limezy** [reply]
I like that approach

**Reply by BradKML** [reply]
Would increase complexity of operations, better off using `alias::` as pages relinking

**Reply by jefftangx** [reply]
This is implemented in 2.0.0-beta.6 https://github.com/athensresearch/athens/releases/tag/v2.0.0-beta.6!

**Comment by muhlinux** [comment]
I think using a pipe would be really clean:
* `[[alias|page]]`
* `((alias|uid))`
